src/SFA.DAS.Tools.Servicebus.Support.Infrastructure.UnitTests/Services/ServiceBus/WhenBatchSendingMessages.cs

```diff
@@ -0,0 +1,74 @@
+﻿using FluentAssertions;
+using Microsoft.Azure.ServiceBus;
+using NUnit.Framework;
+using SFA.DAS.Tools.Servicebus.Support.Domain.Queue;
+using SFA.DAS.Tools.Servicebus.Support.Infrastructure.Services.Batching;
+using System.Collections.Generic;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace SFA.DAS.Tools.Servicebus.Support.Application.UnitTests.Services.MessageService
+{
+    public class WhenBatchSendingMessages
+    {
+        [Test]
+        public async Task ThenMessagesAreBatchedAccordingToMessageSize()
```

Comment:
@DannyGB if I run it it fails, but if I debug it is OK. Any chance you could check if you are seeing the same problem?